test/coordination/test_action_state_machine.rb

```diff
@@ -365,11 +386,16 @@ def action_interface
                         terminates
                         argument :arg
                     end
+                    final_task_m = Roby::Task.new_submodel(name: "Final") do
+                        terminates
+                    end
                     Roby::Actions::Interface.new_submodel do
                         describe("start").returns(start_task_m)
                         define_method(:first) { start_task_m.new }
                         describe("followup").required_arg(:arg, "arg").returns(followup_task_m)
                         define_method(:followup) { |arg: nil| followup_task_m.new(arg: arg) }
+                        describe("final").returns(final_task_m)
+                        define_method(:final) { final_task_m.new }
```

Comment:
I couldn't relate this to the current state registering

Reply:
ah, this is a leftover...I'll remove it